vtexdocs/devportal
# Build/rapidoc code section

#### What is the purpose of this pull request?

To update the curl style, create the structure of the try it section and add two more open API schemas.

#### What problem is this solving?

The style of the curl section doesn't match the expected design in Figma. In addition, the test page needs the try it section that contains the store URL variables and the auth section (with auth keys), after all, Rapidoc-mini use requires the URL params and keys input to generate the correct curl request.

[You can find more information here :book:](https://www.notion.so/vtexhandbook/Estilizar-code-curl-section-da-API-reference-31418fc60aef43fea492baa3ba9fc8af)

**In the future, we'll need to move the Try section to the right side of the page, as the prototype in Figma.**

#### How should this be manually tested?

Check the [open PR](https://github.com/vtexdocs/RapiDoc/pull/9) in our Rapidoc fork.

#### Types of changes
